content/en/tracing/trace_pipeline/ingestion_mechanisms.md

```diff
@@ -57,9 +57,11 @@ All the spans from a trace sampled using the Datadog Agent [automatically comput
 
 For more granular control, use tracing library sampling configuration options:
 - Set a specific **sampling rate to apply to all root services** for the library, overriding the Agent's [default mechanism](#in-the-agent).
-- Set a **sampling rate to apply to specific root services** or for specific span operation names.
```

Comment:
I think this extra phrase was leaving room for interpretation: I could read this as "or one can also target specific non-root services".